tox.ini

```diff
@@ -1,16 +1,17 @@
 # to run for a specific environment, use ``tox -e ENVNAME``
 [tox]
-envlist = py27,py37,py38,py39,py310,pypy3,docs
+envlist = py38,py39,py310,pypy3,docs
+# Note: the 'docs' env creates a 'build' directory which may interfere in strange ways
+# with the other environments.  You might see this when you run the tests in parallel.
+# See https://github.com/collective/icalendar/pull/359#issuecomment-1214150269
```

Comment:
Is that still the case now that you delete it?

Reply:
I think so, when `docs` finishes before one of the other jobs is properly started.